### PR TITLE
Add Publicize scheduling share previews scaffolding

### PR DESCRIPTION
**Pull request by lamosty**
Part of the Publicize Scheduling project (internal ref: p3fqKv-4qj-p2).

After clicking on the "Preview" button in the Share section under a post, we open a dialog in which the user can preview how the share will look like on the connected services.

In this PR, we add Facebook and Twitter share previews scaffolding with design.

preview-linkedin
preview-path
preview-tumblr
preview-gplus
preview-twitter
preview-facebook

## Testing

Go to http://calypso.localhost:3000/devdocs/blocks/sharing-preview-pane and look how the Facebook and Twitter share previews look like. Is it the same as in @iamtakashi's designs? (internal ref: p3fqKv-4qj)

Do not merge until @iamtakashi doesn't add the styling.

### CODE REVIEW

**Comment by iamtakashi** [comment]
Added styles for Facebook and Twitter.

File: client/blocks/sharing-preview-pane/index.jsx [hunk 1]
```diff
@@ -52,7 +52,6 @@ class SharingPreviewPane extends PureComponent {
 				return <FacebookSharePreview
 					title={ seoTitle }
 					url={ get( post, 'URL', '' ) }
-					type="article"
```
Comment (retrofox) [inline client/blocks/sharing-preview-pane/index.jsx]: why did you remove the type?

File: client/components/share/facebook-share-preview/style.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	border-color: #e5e6e9 #dfe0e4 #d0d1d5;
+	border-radius: 3px;
+	color: #1d2129;
+	margin: 0 auto;
+	max-width: 486px;
+	padding: 12px;
+	-webkit-overflow-scrolling: touch;
+}
+
+.facebook-share-preview a {
```
Comment (gwwar) [inline client/components/share/facebook-share-preview/style.scss]: @iamtakashi let's try and follow our css guidelines. In this case, we can add a classname like `facebook-share-preview__link` to each a tag, so we're able to use a selector like: `.facebook-share-preview__link` https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines

@lamosty do you mind updating the markup and selectors?

Comment (gwwar) [inline client/components/share/facebook-share-preview/style.scss]: This is good to land in the feature branch after that's updated

**Comment by gwwar** [comment]
Going to go ahead and land this. We can fix these issues in follow up PRs
